pkg/tools/etcd_helper_watch_test.go

```diff
@@ -207,7 +207,11 @@ func TestWatchEtcdError(t *testing.T) {
 	fakeClient.WatchImmediateError = fmt.Errorf("immediate error")
 	h := EtcdHelper{fakeClient, codec, versioner}
 
-	got := <-h.Watch("/some/key", 4).ResultChan()
+	watching, err := h.Watch("/some/key", 4, Everything)
+	if err != nil {
+		t.Fatalf("Unexpected error: %v", err)
+	}
```

Comment:
Thank you for adding this here-- 

Unfortunately the other places where I didn't make this comment were already correct, I think. You may need to put them back. Sorry, I absolutely should have said that this comment only applied to this one location. :(